README.md

```diff
@@ -17,7 +17,7 @@ Follow the videos in this and next lessons to implement the page block by block:
 - Start writing styles in the `main.scss` file. It is be explained in the `Sass` lesson.
 - **DON'T** try to do it `Pixel Perfect` - implement it the most `simple` way so it looks similar.
 - When done check yourself using the [CHECKLIST](./checklist.md) when finished.
-- Deploy and create a Pull Request with a [DEMO LINK](https://<your_account>.github.io/layout_landing-page/).
+- Deploy and create a Pull Request with a [DEMO LINK](https://SamarashEugene.github.io/layout_landing-page/).
```

Comment:
Add this link to the PR description 
<img width="963" alt="image" src="https://github.com/user-attachments/assets/3d039a9f-4a21-4f76-ab0e-0f79ad7d30c0" />